etc/kayobe/kolla/config/haproxy/services.d/os_capacity.cfg

```diff
@@ -6,7 +6,11 @@ frontend os_capacity_frontend
     option httplog
     option forwardfor
     http-request set-header X-Forwarded-Proto https if { ssl_fc }
-    bind {{ kolla_internal_vip_address }}:9000
+{% if kolla_enable_tls_internal | bool %}
+    bind {{ kolla_internal_vip_address }}:9090 ssl crt /etc/haproxy/certificates/haproxy-internal.pem
```

Comment:
This part of the change needs to be in a PR to stable/2023.1 branch I think.

Reply:
Removed, will readd for antelope.